recipes/appimage-updater-bridge/build.sh

```diff
@@ -0,0 +1,11 @@
+mkdir -p build
+cd build
+
+cmake -G "Ninja" \
+      -D CMAKE_BUILD_TYPE=Release \
+      -D CMAKE_INSTALL_PREFIX:FILEPATH=$PREFIX \
+      -D CMAKE_INSTALL_LIBDIR:FILEPATH=$PREFIX/plugins \
+      -D BUILD_AS_PLUGIN:BOOL=ON \
+      ..
```

Comment:
Is there a `make test` or `ctest` that we can run here?

Reply:
I see some tests included, but it seems like qmake is needed.. @antony-jr are there any tests that make sense to run for this package?

Reply:
@looooo Yes. The tests directory in the repo has the code for tests. But I've only given qmake for compiling the tests. Do you need a CMake for tests?

Reply:
See https://github.com/antony-jr/AppImageUpdaterBridge/blob/master/tests/start_test.sh on how the test are started and executed. The test needs some AppImages to test so I've used the mentioned bash script to download the required AppImages to test.

Reply:
I don't think qmake is available as a conda-forge package. So cmake is definetly the nicer option.

Reply:
I had a look and qmake is included in the qt-package, so this should be no problem. Let's try the script.

Reply:
Okay, If you need CMake though. just ping. But v2.x is being made  I'm making a cmake option for tests in that too.